devlab/utils/clean_run.sh

```diff
@@ -34,7 +34,8 @@ python generate_load.py --clean --env DST
 python generate_load.py --clean --env DST
 ssh root@$DST \
 "mysql -u $dst_mysql_user -p$dst_mysql_password cinder -e \"delete from cinder.volume_admin_metadata;
-delete from cinder.volumes;\""
+delete from cinder.volumes;delete from cinder.reservations;delete from cinder.quota_usages;\""
```

Comment:
You can use all tables from here: https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/cloudferrylib/os/storage/cinder_database.py#L53-L61